File: Board.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows;



namespace Chess
{

    public static class Board
    {
        public static int[] pawn = new int[2]   { 10, 20 };
        public static int[] knight = new int[2] { 11, 21 };
        public static int[] bishop = new int[2] { 12, 22 };
        public static int[] rook = new int[2]   { 13, 23 };
        public static int[] queen = new int[2]  { 14, 24 };
        public static int[] king = new int[2]   { 15, 25 };

        static public int[] white = new int[6] { 10, 11, 12, 13, 14, 15 };
        public static int[] black = new int[6] { 20, 21, 22, 23, 24, 25 };

        public static int[] game = Default();
        public static int[] test = TestBoard();

        public static Dictionary<int, string> pieces = new Dictionary<int, string>()
        {
            {00 , " " },
            {20 , "♟"},
            {21 , "♞"},
            {22 , "♝"},
            {23 , "♜"},
            {24 , "♛"},
            {25 , "♚"},
            {10 , "♙"},
            {11 , "♘"},
            {12 , "♗"},
            {13 , "♖"},
            {14 , "♕"},
            {15 , "♔"},
            {99 , "★" }
        };
        
        public static int[] GetRowCol(int pos)
        {
            int[] output = new int[2] { pos / 8, pos % 8 };
            return output;
        }

        public static int GetPos(int row, int col)
        {
            return row * 8 + col;
        }

        public static int[] Default()
        {
            int[] board = new int[64]  {13, 11, 12, 14, 15, 12, 11, 13,
                                        10, 10, 10, 10, 10, 10, 10, 10,
                                        00, 00, 00, 00, 00, 00, 00, 00,
                                        00, 00, 00, 00, 00, 00, 00, 00,
                                        00, 00, 00, 00, 00, 00, 00, 00,
                                        00, 00, 00, 00, 00, 00, 00, 00,
                                        20, 20, 20, 20, 20, 20, 20, 20,
                                        23, 21, 22, 24, 25, 22, 21, 23};
            return board;
        }

        public static int[] TestBoard()
        {
            int[] board = new int[64]  {00, 00, 00, 00, 00, 00, 00, 00,
                                        10, 00, 00, 00, 00, 00, 00, 00,
                                        00, 00, 00, 00, 00, 00, 00, 00,
                                        00, 00, 00, 00, 00, 00, 00, 00,
                                        00, 00, 00, 00, 00, 00, 00, 00,
                                        00, 00, 00, 00, 00, 00, 00, 00,
                                        00, 00, 00, 20, 00, 00, 00, 00,
                                        23, 21, 22, 24, 25, 22, 21, 23};
            return board;
        }
    }



}

File: Console Application/board.py
pawn = [10, 20]
knight = [11, 21]
bishop = [12, 22]
rook = [13, 23]
queen = [14, 24]
king = [15, 25]

white = [10, 11, 12, 13, 14, 15]
black = [20, 21, 22, 23, 24, 25]

pieces = {20 : "♟", 21 : "♞", 22 : "♝", 23 : "♜", 24 : "♛", 25 : "♚", 10: "♙", 11 : "♘", 12 : "♗", 13 : "♖", 14 : "♕", 15 : "♔", "test" : "★"}

row0 = range(0, 8)
row1 = range(8, 16)
row2 = range(16, 24)
row3 = range(24, 32)
row4 = range(32, 40)
row5 = range(40, 48)
row6 = range(48, 56)
row7 = range(56, 64)

rows = [row0, row1, row2, row3, row4, row5, row6, row7]


def GetRowCol(position):

    output_row = position // 8
    output_col = position % 8
    return output_row, output_col


def GetPosition(row, col):

    return row*8 + col

def default():

    return [13, 11, 12, 14, 15, 12, 11, 13] + [10] * 8 + [0] * 32 + [20] * 8 + [23, 21, 22, 24, 25, 22, 21, 23]


def PrintBoard(board):

    output_rows = []
    j = 7
    for row in rows[::-1]:
        output = ""
        for i in range(len(board)):
            if i in row:
                row_index = rows.index(row)
                if board[i] == 0:
                    if row_index % 2 == 0:
                        if i % 2 == 0:
                            output += " - "
                        else:
                            output += "   "
                    else:
                        if i % 2 == 0:
                            output += "   "
                        else:
                            output += " - "
                else:
                    output += " " + pieces[board[i]] + " "

        output = str(j) + " ║ " + output + " ║ "
        output_rows.append(output)
        j -= 1

    print("  ╔══════════════════════════╗")
    for row in output_rows:
        print(row)
    print("  ╚══════════════════════════╝")
    print("     0  1  2  3  4  5  6  7 ")